ethereum/package.json

```diff
@@ -18,7 +18,7 @@
   },
   "scripts": {
     "build:core": "truffle compile",
-    "build:forge": "if [ \"$DEV\" = \"True\" ]; then forge build; else forge build --via-ir; fi",
+    "build:forge": "forge build",
```

Comment:
This (together with the foundry.toml file) changes the default behaviour from building `via-ir` to building without it. Also, previously to disable this behaviour, one would have to run
```sh
export DEV=True
```
but now, to *enabled* this behaviour, one has to run
```sh
export FOUNDRY_PROFILE=production
```

This is fine, but might be worth documenting in the README?

Reply:
The `--via-ir` flag as really only intended for the relayer contracts. Since we don't currently build the base contracts with that flag, I removed it from the command. If we decide to "upgrade" to building with IR in the future, it can be re-added.